packages/coreutils/src/poll.ts

```diff
@@ -364,12 +364,10 @@ export class Poll<T = any, U = any> implements IDisposable, IPoll<T, U> {
       return;
     }
 
-    const pending = this._tick;
-
     // The `when` promise in the constructor options acts as a gate.
     if (this.state.phase === 'constructed') {
       if (next.phase !== 'when-rejected' && next.phase !== 'when-resolved') {
-        await pending.promise;
+        await this.tick;
```

Comment:
This part is equivalent to the old code, right? `this.tick` is `this._tick.promise`.

Reply:
This part is equivalent, yes.